### ISSUE TITLE
Validation loss

### ISSUE THREAD

**Issue by viethoang303** [juhongm999/hsnet#26]
Hello authors and everyone. 

I trained HSnet on my own dataset about echocardigraphy images, but validation loss fluctuates a lot and model converge early. I trained model with learning rate 1e-3 and I looked Fig 9 in paper about this model, validation loss also take 10-30 epochs to achieve higher validation IoU

![Validation-loss-hsnet_1shot_on_3C](https://user-images.githubusercontent.com/48119382/166435433-c5953966-4ab6-4b65-8b67-9fb698ef7cf4.png)

So, what problem I got. Please help me.

Thank

**Comment by juhongm999** [comment]
Perhaps the size of your dataset is too small, making the model converge quickly.
How many images are there in your dataset?
